yb-voyager/cmd/import.go

```diff
@@ -208,11 +208,11 @@ func registerImportDataCommonFlags(cmd *cobra.Command) {
 		defaultbatchSize = int64(DEFAULT_BATCH_SIZE_ORACLE)
 	}
 	cmd.Flags().Int64Var(&batchSize, "batch-size", defaultbatchSize,
-		"maximum number of rows in each batch generated during import of snapshot.")
-	defaultParallelismMsg := "(default - oracle: 16)"
-	if cmd.CommandPath() == "yb-voyager import data" {
-		defaultParallelismMsg = "By default, voyager will try if it can determine the total number of cores N and use N/2 as parallel jobs. " +
-			"Otherwise, it fall back to using twice the number of nodes in the cluster."
+		"Size of batches in the number of rows generated for ingestion during import.")
+	defaultParallelismMsg := "By default, voyager will try if it can determine the total number of cores N and use N/2 as parallel jobs. " +
+		"Otherwise, it fall back to using twice the number of nodes in the cluster."
+	if cmd.CommandPath() == "yb-voyager fall-back setup" || cmd.CommandPath() == "yb-voyager fall-forward setup" {
```

Comment:
Switched the logic back to check for fall-back/fall-forward because with current logic, import data file was also showing "default-oracle:16"